alephmarcreader/tests/test_MarcXMLReader.py

```diff
@@ -48,6 +48,24 @@ def test_date(self):
 
         self.assertEqual(date[0], u'1734.03.12')
 
+    def test_get_standardized_date(self):
+        # Test with precise date
+        marcxml_rd = AlephMarcXMLReader('alephmarcreader/tests/sample_data/MarcXML/000055275.xml')
+
+        std_date = marcxml_rd.get_standardized_date()
+        self.assertEqual(len(std_date), 1)
+        self.assertEqual(std_date[0].start_span, u'1734-03-12')
+        self.assertEqual(std_date[0].end_span, False)
+        self.assertEqual(std_date[0].get_standardized_date_string_KNORA(), u'1734-03-12')
```

Comment:
I think you forgot to check for the ISO date.

Reply:
yes... long day, yesterday. ;)